Symfony/CS/Tokenizer/TokensAnalyzer.php

```diff
@@ -210,6 +210,71 @@ public function isArrayMultiLine($index)
         return false;
     }
 
+    public function getFunctionAttributes($index)
```

Comment:
>what I dislike is not attributes but function - it is method.

Thanks :)
I think this should do it:
https://github.com/SpacePossum/PHP-CS-Fixer/commit/04cc5348bee143648c9439937e12879009a49779